vsphere_server_plugin/server.py

```diff
@@ -333,6 +333,83 @@ def create_new_server(server_client,
     return server_obj
 
 
+@op
+@with_server_client
+def start(server_client,
```

Comment:
the name operation should be create ? I see there is already start operation in this file.